.ci/docker/requirements-ci.txt

```diff
@@ -362,6 +362,7 @@ pwlf==2.2.1 ; python_version >= "3.8"
 # To build PyTorch itself
 astunparse
 PyYAML
+pyzstd
```

Comment:
This is NOT a build dependency, only an optional runtime dependency?

Reply:
correct.